src/System.Net.Mail/tests/Functional/SmtpClientTest.cs

```diff
@@ -363,5 +363,51 @@ public async Task TestCredentialsCopyInAsyncContext()
                 server.Stop();
             }
         }
+
+        [Fact]
+        public void TestSmtpDeliveryFormatSevenBit()
+        {
+            string sendSubject = "", sendAsyncSubject = "";
+
+            //Run twice to get the results of synchronous and asynchronous methods
```

Comment:
You can use attribute [Theory], and pass in a parameter into this test method, which can be named as  `useAsyncSend` = true/false. Based on that boolean, a sync/async send() can be performed.

If you use this approach, the test name can be changed to `SendMail_DeliveryFormatSevenBit_SubjectEncoded`.